vector-im/element-android
# [Poll] Render past polls list of a room (PSG-1029)

## Type of change


## Content


Render a mocked list of ended polls in the poll history screen.

## Motivation and context


Continuation of #7867 
Part of #7864 

## Screenshots / GIFs





<img src="https://user-images.githubusercontent.com/46314705/210535330-5d1deebb-3a39-49da-b5e7-4d038e71d0a7.png" width=300 /> <img src="https://user-images.githubusercontent.com/46314705/210535335-be0ac7f1-154a-49c5-a965-e8953a21fc1b.png" width=300 />

## Tests



- Go to the profile screen of a room
- Press the "Poll history" entry
- Check there is tab displaying a mocked list of ended polls

## Tested devices

- OS version(s): Android 11

## Checklist
